src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -251,161 +253,214 @@ public InventoryType getType()
 	@Override
 	public int getMaxStackSize()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return 64;
 	}
 	
 	@Override
-	public void setMaxStackSize(int size)
-	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
-		
-	}
+	public void setMaxStackSize(int size) {}
```

Comment:
Doesn't feel like a valid implementation

Reply:
i think every inventory has a 64 max stack size so i've assumed that